installer/pyinstaller/samcli-nightly.spec

```diff
@@ -0,0 +1,29 @@
+# -*- mode: python -*-
+block_cipher = None
+exe_name = 'sam-nightly'
```

Comment:
can the exe_name be parameterized too?

Reply:
These files are load as parameter of pyinstaller, so I feel that loading different file is a better way and not easy to make mistake.

Reply:
I have updated the install script, the name here can be replaced by script now.